src/index.js

```diff
@@ -1091,7 +1094,7 @@ class IString {
      * @return {IString} the normalize version of the string
      */
     normalize(form) {
-        return typeof(this.str.normalize) === 'function' ? new IString(this.str.normalize(form)) : this;
+        return typeof(this.str.normalize) === 'function' ? new IString(this.str.normalize(form), {locale: this.locale}) : this;
```

Comment:
the most changes are added locale parameter. should locale parameter is added all the time? i remember there was no problem in using ilib as a one time setlocale call

Reply:
In ilib, IString.setLocale was asynchronous because it needed to load the plurals.json, which meant all sorts of odd things had to be asynchronous in order to call it, or the locale had to be not set. As such, the IString was not immutable. In the module-based ilib, all the asynchronous behaviour is in the constructor only and it is an immutable class. To achieve that, we have to pass the locale at construction time.